npins/sources.json

```diff
@@ -1,5 +1,16 @@
 {
   "pins": {
+    "flake-compat": {
```

Comment:
Now that it is pinned we will need to replace the code in the flake loader:

https://github.com/nilla-nix/nilla/blob/0ffd40e1811c5cec62ffe79efdcf2bc601cfdf38/modules/loaders/flake.nix#L3